src/Browscap/Coverage/Processor.php

```diff
@@ -0,0 +1,571 @@
+<?php
+/**
+ * Copyright (c) 1998-2017 Browser Capabilities Project
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Refer to the LICENSE file distributed with this package.
+ *
+ * @category   Browscap
+ * @copyright  1998-2017 Browser Capabilities Project
+ * @license    MIT
+ */
+
+namespace Browscap\Coverage;
+
+use Seld\JsonLint\Lexer;
+
+/**
+ * Class Processor
+ *
+ * @category   Browscap
+ * @author     Jay Klehr <jay.klehr@gmail.com>
+ */
+class Processor
+{
+    /**
+     * @var string
+     */
+    private $resourceDir;
+
+    /**
+     * These are available in the JSON Parser part of the JSON Lint package we're using
+     * to Lexically analyze the JSON file, but they're a private property so can't re-use them directly
+     *
+     * @var array
+     */
+    private $symbols = [
+        'error' => 2,
+        'JSONString' => 3,
+        'STRING' => 4,
+        'JSONNumber' => 5,
+        'NUMBER' => 6,
+        'JSONNullLiteral' => 7,
+        'NULL' => 8,
+        'JSONBooleanLiteral' => 9,
+        'TRUE' => 10,
+        'FALSE' => 11,
+        'JSONText' => 12,
+        'JSONValue' => 13,
+        'EOF' => 14,
+        'JSONObject' => 15,
+        'JSONArray' => 16,
+        '{' => 17,
+        '}' => 18,
+        'JSONMemberList' => 19,
+        'JSONMember' => 20,
+        ':' => 21,
+        ',' => 22,
+        '[' => 23,
+        ']' => 24,
+        'JSONElementList' => 25,
+        '$accept' => 0,
+        '$end' => 1,
+    ];
+
+    /**
+     * @var array
```

Comment:
This one ends up being the big array that is json_encoded and written to the file, it ends up being a multi-dimensional array (with the keys seen below; `fnMap`, `statementMap`, etc...).